python/hsfs/core/delta_engine.py

```diff
@@ -168,6 +278,51 @@ def _generate_merge_query(self, source_alias, updates_alias):
         megrge_query_str = " AND ".join(merge_query_list)
         return megrge_query_str
 
+    @staticmethod
+    def _get_last_commit_metadata_delta_rs(base_path):
```

Comment:
maybe we could change it so that there would be less duplication with `def _get_last_commit_metadata(`?

Reply:
This needs more testing but to me logfiles seem different. for example in spark Delta there is camel case names but in detla-rs snake case names: `num_target_rows_inserted`